rdflib/query.py

```diff
@@ -57,7 +59,7 @@ class update method.
     def __init__(self, graph):
         pass
 
-    def update(self, strOrQuery, initBindings={}, initNs={}):
+    def update(self, str_or_query, initBindings={}, initNs={}):  # noqa: N803
```

Comment:
I think maybe you intended to have:

```suggestion
    def update(self, strOrQuery, initBindings={}, initNs={}):  # noqa: N803
```

But even if not, I think it is best to keep the old `strOrQuery`.

Alternatively, we can retain it as a keyword only arg so that if someone passes it we can both log a deprecation warning and still do the right thing.

```suggestion
    def update(self, str_or_query = None, initBindings={}, initNs={}, *, strOrQuery = None):  # noqa: N803
```

But I think for now just keeping it as is with noqa is simplest.

Reply:
Ah-ha, even with the fix to `str_or_query`, the `# noqa: N803` is still required for the `initBindings` and `initNS` violations, I didn't spot my typos until _far_ to late :frowning_face:

Reply:
Reverted change to snakecase.